app/enterprise/1.3-x/developer-portal/helpers/cli.md

```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
 
 The easiest way to start is by cloning the [portal-templates repo][templates] master branch locally.
 
-Then edit `workspaces/default/cli.conf.yaml` to set workspace `name` and `rbac_token` to match your setup.
+Then edit `workspaces/default/cli.conf.yaml` to set `kong_admin_uri` and `kong_admin_token` to match your setup.
 
 Make sure Kong is running and portal is on:
 
 Now from root folder of the templates repo you can run:
 
-```portal [-h,--help] [--config PATH] [-v,--verbose] <command>```
+```portal <command> <workspace>```
 
 Where `<command>` is one of:
+ - `config`    Output or change configuration of the portal on the given
+ - `deploy`    Deploy changes made locally under the given workspace upstream.
+ - `disable`   Enable the portal on the given workspace.
+ - `enable`    Enable the portal on the given workspace.
+ - `fetch`     Fetches content and themes from the given workspace.
+ - `serve`     Run the portal of a given workspace locally.
+ - `wipe`      Deletes all content and themes from upstream workspace
 
-* `config`   Output or change configuration of the portal on the given
-* `workspace`, locally.
-* `deploy`   Deploy changes made locally under the given workspace upstream.
-* `disable`  Enable the portal on the given workspace.
-* `enable`   Enable the portal on the given workspace.
-* `fetch`    Fetches content and themes from the given workspace.
-* `serve`    Run the portal of a given workspace locally.
-* `wipe`     Deletes all content and themes from upstream workspace
+ Where `<workspace>` indicates the directory/workspace pairing you would like to operate on.
 
-Add `--watch` to make changes reactive
+### For `deploy`
+- Add `-W` or `--watch` to make changes reactive
+- Add `-P` or `--preserve` to avoid deleting files upstream that you do not have locally
 
+### For `fetch`
+- Add `-K` or `--keep-encode` to make keep binary assets as base64 encoded strings locally
```

Comment:
Suggest removing "make" before "keep", or other way around, but not "make keep"
`- Add `-K` or `--keep-encode` to keep binary assets as base64 encoded strings locally`